src/main/scala/org/ergoplatform/http/api/ErgoBaseApiRoute.scala

```diff
@@ -22,6 +26,8 @@ trait ErgoBaseApiRoute extends ApiRoute with ApiCodecs {
 
   implicit val ec: ExecutionContextExecutor = context.dispatcher
 
+  implicit val ae: ErgoAddressEncoder = null
```

Comment:
this null can be eliminated by moving to BlockchainApiRoute :

```
  val ae: ErgoAddressEncoder = ergoSettings.chainSettings.addressEncoder

  val ergoAddress: Directive1[ErgoAddress] = entity(as[String]).flatMap(handleErgoAddress)

  private def handleErgoAddress(value: String): Directive1[ErgoAddress] = {
    ae.fromString(value) match {
      case Success(addr) => provide(addr)
      case _ => reject(ValidationRejection("Wrong address format"))
    }
  }
```

Reply:
done